go.mod

```diff
@@ -76,3 +51,10 @@ require (
 )
 
 replace github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 => github.com/m4ns0ur/httpcache v0.0.0-20200426190423-1040e2e8823f
+
+replace (
+	github.com/devopsfaith/krakend-amqp v1.4.0 => github.com/openrm/krakend-amqp v0.0.0-20210526165636-5cc9045b5fa6
+	github.com/devopsfaith/krakend-cel v1.4.0 => github.com/openrm/krakend-cel v0.0.0-20210719101150-9ec8a72804f1
+	github.com/devopsfaith/krakend-jose v1.4.0 => github.com/openrm/krakend-jose v0.0.0-20210719112350-52718b067cbd
+	github.com/devopsfaith/krakend-martian v1.4.0 => github.com/openrm/krakend-martian v0.0.0-20210718024057-b5801065122d
+)
```

Comment:
Instead of replacing the module import declarations, I used this direction this time. Could minimize the change to the upstream, working well so far

Reply:
good idea, it will also help to replace it when doing some local testing